### PR TITLE
focus specific device in and out

### PR DESCRIPTION
**Pull request by esprush**

### CODE REVIEW

File: desktop-app/app/actions/browser.js [hunk 1]
```diff
@@ -284,7 +292,7 @@ export function changeDrawerOpenState(newOpenState) {
   };
 }
 
-export function setPreviewLayout(newLayout) {
+export function setPreviewLayout(newLayout, focusedDeviceId) {
```
Comment (manojVivek) [inline desktop-app/app/actions/browser.js:295]: Can you remove the `focusedDeviceId` arg ?

File: desktop-app/app/components/DevicesPreviewer/index.js [hunk 2]
```diff
@@ -25,12 +25,47 @@ export default function DevicesPreviewer(props) {
...
+  };
+
   return (
     <div className={cx(styles.container)}>
       {layout === INDIVIDUAL_LAYOUT && (
-        <Tabs onSelect={changeTab}>
+        <Tabs onSelect={onTabClick} selectedIndex={focusedDeviceIndex}>
           <TabList>
-            {devices
+            {/* {devices
```
Comment (manojVivek) [inline desktop-app/app/components/DevicesPreviewer/index.js:68]: Can you pls remove the commented code?

File: desktop-app/app/reducers/browser.js [hunk 3]
```diff
@@ -321,6 +330,14 @@ export default function browser(
           : device
       );
       return {...state, devices: newDevicesList};
+    // case DEVICE_FOCUS:
```
Comment (manojVivek) [inline desktop-app/app/reducers/browser.js:333]: Can you please delete this too?

File: desktop-app/app/actions/browser.js (hunk 1 above)
Comment (esprush) [reply desktop-app/app/actions/browser.js]: removed

File: desktop-app/app/components/DevicesPreviewer/index.js (hunk 2 above)
Comment (esprush) [reply desktop-app/app/components/DevicesPreviewer/index.js]: done Manoj

File: desktop-app/app/reducers/browser.js (hunk 3 above)
Comment (esprush) [reply desktop-app/app/reducers/browser.js]: done Manoj

**Pull request merged**